bokeh/io.py

```diff
@@ -340,6 +350,9 @@ def _show_notebook_with_state(obj, state, notebook_handle):
             state.last_comms_handle = handle
             return handle
 
+def _show_zeppelin(obj):
```

Comment:
suggest `show_zeppelin_with_state` to keep a consistent interface. In fact, I'd have it accept all the same args, including `notebook_handle`, then that's the place that can raise a "Zeppelin does not support notebook handles" error if `notebook_handle` is true. Also `_show_notebook_with_state` should be renamed `_show_jupyter_with_state`